contrib/golang/filters/http/source/processor_state.h

```diff
@@ -164,6 +164,7 @@ class ProcessorState : public Logger::Loggable<Logger::Id::http>, NonCopyable {
                               Grpc::Status::GrpcStatus grpc_status, absl::string_view details) PURE;
 
   const StreamInfo::StreamInfo& streamInfo() const { return filter_callbacks_->streamInfo(); }
+  StreamInfo::StreamInfo& streamInfo() { return filter_callbacks_->streamInfo(); }
```

Comment:
does remove line 166 could be better? I'm not sure about it.

Reply:
I saw other places like `quic_filter_manager_connection_impl.h` implement both function signatures. that way you can use `streamInfo` in both `const` and not `const` functions

Reply:
okay, I see, thank you!